internal/utils/test.go

```diff
@@ -105,3 +105,16 @@ func GetDataSourcesLength(state *terraform.State, tfVarName, dataSourceName stri
 
 	return instanceState, numAttribute, nil
 }
+
+var OrganizationRoles = []string{string(iam.ORGANIZATIONBILLINGADMIN), string(iam.ORGANIZATIONMEMBER), string(iam.ORGANIZATIONOWNER)}
+var WorkspaceRoles = []string{string(iam.WORKSPACEACCESSOR), string(iam.WORKSPACEAUTHOR), string(iam.WORKSPACEMEMBER), string(iam.WORKSPACEOWNER), string(iam.WORKSPACEOPERATOR)}
+var DeploymentRoles = []string{"DEPLOYMENT_ADMIN"}
```

Comment:
we should have this inside the function and have the function
be something like `CheckRole(role string, scopeType string)`